core/class.go

```diff
@@ -56,7 +56,28 @@ type Cairo1Class struct {
 	Program         []*felt.Felt
 	ProgramHash     *felt.Felt
 	SemanticVersion string
-	Compiled        json.RawMessage
+	Compiled        CompiledClass
+}
+
+type CompiledClass struct {
+	Bytecode        []*felt.Felt        `json:"bytecode"`
```

Comment:
We don't need json tags in the core package.

Reply:
I had to include JSON tags here for the marshaling happening on the [VM class](https://github.com/NethermindEth/juno/blob/d0e07f0f0cac1e0c656a55de87350a375cb1dd33/vm/class.go#L32), as the VM expects `CompiledClass` as json raw.

Reply:
Is it possible to use the feeder type for marshalling, similar to the Cairo0 compiled class in that example?

If not, we should have a special type in `vm/class.go` for serialization/deserialization in that case.

Reply:
Yes, I guess we can use the feeder type. I will go for it.